src/test/java/duke/task/TaskListTest.java

```diff
@@ -0,0 +1,35 @@
+package duke.task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class TaskListTest {
+    private static final TaskStub TASK_ONE = new TaskStub("task 1", false);
+    private static final TaskStub TASK_TWO = new TaskStub("task 2", false);
+    private static final TaskStub TASK_TWO_COMPLETE = new TaskStub("task 2", true);
+    private static final TaskStub TASK_THREE = new TaskStub("task 3", true);
+
+    @Test
+    public void testTaskList() {
+        TaskList taskList = new TaskList();
+        assertEquals(taskList.getCurrCapacity(), 0);
```

Comment:
Good use of assertEquals! I really like how your tests are comprehensive as well! Great job 😄